src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/commands.py

```diff
@@ -7,24 +7,44 @@
 from ._exception_handler import consumption_exception_handler
 from ._validators import validate_both_start_end_dates
 from ._validators import validate_reservations_summaries
 from ._validators import validate_reservations_details
+from ._validators import validate_pricesheet
+from ._validators import validate_usage_bp_inputs
 
 
 def load_command_table(self, _):
     with self.command_group('consumption usage') as g:
         g.custom_command('list', 'cli_consumption_list_usage', transform=transform_usage_list_output,
                          exception_handler=consumption_exception_handler, validator=validate_both_start_end_dates, client_factory=usage_details_mgmt_client_factory)
 
+        g.custom_command('billing period list', 'cli_consumption_list_usage_by_billing_period', transform=transform_usage_list_output,
+                         exception_handler=consumption_exception_handler, validator=validate_usage_bp_inputs, client_factory=usage_details_mgmt_client_factory)
+
     with self.command_group('consumption reservations summaries') as s:
         s.custom_command('list', 'cli_consumption_list_reservations_summaries', transform=transform_reservation_summaries_list_output,
                          exception_handler=consumption_exception_handler, validator=validate_reservations_summaries, client_factory=reservations_summaries_mgmt_client_factory)
 
+        s.custom_command('reservation id list', 'cli_consumption_list_reservations_summaries_reservation_id', transform=transform_reservation_summaries_list_output,
+                         exception_handler=consumption_exception_handler, validator=validate_reservations_summaries, client_factory=reservations_summaries_mgmt_client_factory)
+
     with self.command_group('consumption reservations details') as d:
         d.custom_command('list', 'cli_consumption_list_reservations_details', transform=transform_reservation_details_list_output,
                          exception_handler=consumption_exception_handler, validator=validate_reservations_details, client_factory=reservations_details_mgmt_client_factory)
+
+        d.custom_command('reservation id list', 'cli_consumption_list_reservations_details_by_reservation_id', transform=transform_reservation_details_list_output,
+                         exception_handler=consumption_exception_handler, validator=validate_reservations_details, client_factory=reservations_details_mgmt_client_factory)
+
+    with self.command_group('consumption pricesheet') as p:
+        p.custom_command('get', 'cli_consumption_list_pricesheet_get', transform=transform_pricesheet_get_output,
+                         exception_handler=consumption_exception_handler, validator=None, client_factory=pricesheet_mgmt_client_factory)
+
+        p.custom_command('billing period get', 'cli_consumption_list_pricesheet_by_billing_period_get', transform=transform_pricesheet_get_output,
```

Comment:
`get` is not an acceptable command name in the CLI.  https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#command-naming-and-behavior-guidance

The correct name would be `show`.